src/utils.libsonnet

```diff
@@ -9,5 +9,7 @@
     encodeQuery(params)::
       local fmtParam(p) = '%s=%s' % [self.escapeString(p), self.escapeString(params[p])];
       std.join('&', std.map(fmtParam, std.objectFields(params))),
+
+    delay(ms):: 'setTimeout(function(){}, %s);' % std.manifestJson(ms)
```

Comment:
please remove this from utils, since we agreed that it was more of a script than a jsonnet utility

Reply:
I just created a new commit. Please check again.